### ISSUE TITLE
Play speed depends on bitrate and channels

### ISSUE THREAD

**Issue by joshvazquez** [nstbayless/flyweb-mp3#23]
A 48 kHz mono MP3 plays at nearly twice the normal speed because the player is hardcoded to expect stereo 44.1 kHz. However, the progress bar shows the progression for normal speed, and the web server crashes when the end of the song is reached before the progress bar reaches the end.

```
events.js:141
      throw er; // Unhandled 'error' event
      ^

Error: write() failed: 348
    at onwrite (/home/josh/Documents/flyweb/flyweb-mp3/node_modules/speaker/index.js:269:12)
```

A stereo 48 kHz file plays 10% slower but does not cause the server to crash. The progress bar shows the time elapsed exceeding the duration of the song when the duration is passed.

@wchae @sinoforever both of you have worked on audio playing so not sure who to assign this to.

**Comment by sinoforever** [comment]
Yea we have to support different bitrates, the progress bar is sort of hardcoded as well. I'll work on it.

**Comment by sinoforever** [comment]
fixed in da053d6
